docs/source/guides/tools/rpakpilotmodding.rst

```diff
@@ -0,0 +1,245 @@
+.. _importpilotmodel:
+
+Rpak Pilot Model Importing
+=====
+
+Tools:
+^^^^^
+
+Download each tool. 
+
+.. _Blender: https://www.blender.org/
+
+.. _Blender Source Tools: http://steamreview.org/BlenderSourceTools/
+
+.. _Titanfall VPK Tools: https://retryy.gitbook.io/tf2/Wiki/Tools/vpk-tools
+
+.. _Crowbar: https://steamcommunity.com/groups/CrowbarTool
+
+.. _Legion+: https://github.com/r-ex/LegionPlus
+
+.. _RePak: https://github.com/r-ex/RePak
+
+.. _mdlshit: https://github.com/headassbtw/mdlshit-binaries
+
+.. _Text Editors: https://retryy.gitbook.io/tf2/Wiki/Tools/general-pc-tools#text-editing
+
+
+
+- `Text Editors`_ - Just a text editor
+
+- `Titanfall VPK Tools`_ - VPK viewer, extractor, and repacker.
+            "Titanfall VPK Tool" is the vpk tool used. Made by cra0.
+  
+- `Crowbar`_ - Decompile, recompile, and a ton of other model and source asset stuff
```

Comment:
I would prefer if a different vpk tool was recommended here, as they are generally considered better these days (Crowbar can do vpk extraction for titanfall 2 properly, so you could just recommend Crowbar for it to avoid having too many required tools)

This change would however require a rewriting/rewording of the section dedicated to extracting the model from the vpk

Reply:
I never recommended a specific vpk tool.

I will take another look in VPK tools to see what is the overall best, but I just continued with cra0's VPKTool because it's popular use in tutorials, it's ability to unpack AND repack. 

If I choose Crowbar, I'll rewrite the necessary content.